### ISSUE TITLE
This integration is NOT for the normal Ecopower contracts

### ISSUE THREAD

**Issue by lennie-BE** [infradom/ecopower_dynamic_grid_prices#4]
It seems usefull to me to clarify more clearly in the documentation that this integration is only usefull for a limited number of Ecopower clients in a pilot project that use dynamic grid prices.

I know that this is exactly the name of this integration but never the less some mixup with the new price calculation since november 2022 (semi fixed + semi variable) for the regular clients is possible. It least I understood it wrong en asked Ecopower for access to their API to find out it would't do what I expected it to do.

**Comment by infradom** [comment]
Thanks for the feedback. I added a warning in the README

**Comment by lennie-BE** [comment]
Well done! Thanks!

**Comment by thibaultmol** [comment]
I have a normal ecopower contract. But I would like to see the dynamic tariffs in home assistant already, just to get a 'feel for it'.
Guess there's no problem in adding it then?

**Comment by infradom** [comment]
> Guess there's no problem in adding it then?
No, this  changes nothing to the Home Assistant, just adds some entities that will be there for information only

**Comment by thibaultmol** [comment]
Yeah, needing API key's is something I was afraid of /assumed. Guess I could contact support to see if they could provide them to me, even though i'm not in the test project. Probably not. But who knows

**Comment by infradom** [comment]
I would recommend to use the Nordpol integration. 
Nordpol can be used as backup source for my integration (not sure if it will work without the API keys).. It has been a long time since I developed this, so I need to look into the code for a more accurate answer

**Comment by thibaultmol** [comment]
Thx for the recommend! Nordpol works perfectly!

**Comment by TheGroundZero** [comment]
Any idea on what the `additional_costs` config for Nordpool would be to correctly calculate the Ecopower prices?

Price for grid import is `1,02 * EPEX DA + 4 euro/MWh` but then there's also the contributions for GSC and WKK as well as region-based tarrifs (nettarieven) and levies (heffingen), and VAT (6%)

https://cdn.nimbu.io/s/4tn7vz5/assets/1695216932291/dynamisch_tariefkaart_ecopower_september2023.pdf

**Comment by infradom** [comment]
The default values should include all the charges that are calculated on a per kWh basis (see your bill). It may depend slightly on the region (I am Imewo region. (or maybe Fluvius West).
<img width="396" alt="image" src="https://github.com/infradom/ecopower_dynamic_grid_prices/assets/11804014/0879a0cb-48b3-415e-b909-fb1cb73235a2">

prijs_consumptie = (ENTSOE * Factor_A + Factor_B) * BTW_consumptie
prijs_injectie = (ENTSOE * Factor_C + Factor_D) * BTW_injectie
Factor_A = 0.00102
Factor B = 0.004 + 0.0378329  + 0.0019 + 0.0475 + 0.0165 + 0.0028 ( kost ecopower + fluvius afname + bijdrage energie + bijz accijns + kosten groene stroom + WKK)
Factor_C = 0.00085
Factor_D = 0.004
BTW_consumptie = 1.06
BTW_injectie = 1.00

**Comment by infradom** [comment]
Sorry, I was confused, and answered for the wrong ingegration.
I hava also created a new integration that can be used for ecopower, but also for other companies: See https://github.com/infradom/dynamic_grid_prices
The information given above is for this integration
